integration-tests/README.md

```diff
@@ -68,16 +68,48 @@ can either be 8 or 11.
 Druid's configuration (using Docker) can be overrided by providing -Doverride.config.path=<PATH_TO_FILE>. 
 The file must contain one property per line, the key must start with `druid_` and the format should be snake case. 
 
+## Running tests by docker-compose + mvn
+
+Build druid-cluster, druid-hadoop docker images. From root module run maven command:
+```
+mvn clean install -pl integration-tests -P integration-tests -Ddocker.run.skip=true -Dmaven.test.skip=true
+```
+
+Run druid cluster by docker-compose:
+```
+docker-compose -f integration-tests/docker/docker-compose.yml up
+```
+
+Run integration tests:
```

Comment:
I think this is not needed as this is still the same as the instruction above. Since we already have the instruction to run the test in ## Running tests and instruction on using -Ddocker.x.skip, I think this is redundant.

Reply:
removed

Reply:
LGTM